### PR TITLE
task was added

### PR DESCRIPTION
**Pull request by mgrinko**

### CODE REVIEW

**Comment by mgrinko** [comment]
@danmysak 
An input string can have lower or upper case (it does not matter)
I think `invertColor` tells exactly that the result is an inverted color

File: src/invertColor.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * Write a function accepting 6 digit HEX color with a leading #
+ * and returning inverted color
+ *
+ * invertColor('#000000') === '#FFFFFF'
+ * invertColor('#DDEEAA') === '#221155'
+ * invertColor('#012345') === '#FEDCBA'
```
Comment (roman-mate) [inline src/invertColor.js]: Do we want this function to support shorthand colors too?

File: src/invertColor.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const invertColor = require('./invertColor');
+
+test('#000000', () => {
```
Comment (roman-mate) [inline src/invertColor.test.js]: Each task case should be testing different scenarios. The description should explain which scenario is being tested. 

Here is an example of good descriptions: https://github.com/mate-academy/js_capitals-first/blob/master/src/capitalsFirst.test.js

File: src/invertColor.js (hunk 1 above)
Comment (mgrinko) [reply src/invertColor.js]: I think we could start with a simple version (just)

File: src/invertColor.test.js (hunk 2 above)
Comment (mgrinko) [reply src/invertColor.test.js]: That's why I wanted guys to automate it 
Not to think about all possible cases

Comment (roman-mate) [reply src/invertColor.test.js]: C'mon you are a senior engineer. This takes 10 mins.

Comment (mgrinko) [reply src/invertColor.test.js]: done

**Comment by mgrinko** [comment]
@roman-mate @danmysak 
Any other comments?

**Pull request merged**